test/lib/processOpenAPI/parseOpenapi.circularRef.test.ts

```
@@ -0,0 +1,17 @@
+import { describe, expect, it } from 'vitest'
+import { getSchemaUi } from '../../../src/lib/parser/getSchemaUi'
+import { specWithCircularRef } from '../../testsConstants'
+import { parseOpenapi } from '../../../src/lib/parser/parseOpenapi'
```

ESLint

[error] Expected "../../../src/lib/parser/parseOpenapi" to come before "../../testsConstants".

(perfectionist/sort-imports)